.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release Che Machine Exec
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        description: 'The version that is going to be released. Should be in format 7.x.y'
+        required: true
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create Release
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+          docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
I think you can use this on action: https://github.com/eclipse/che-machine-exec/blob/master/.github/workflows/gh_actions_push.yaml#L27 for registry login

Reply:
good idea, also did that for docker login (to avoid rate limits). See additional commits.